assets/robotest/resources/app.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: bundle.gravitational.io/v2
```

Comment:
"bundle" is the legacy version/kind, let's use new header:

```yaml
apiVersion: cluster.gravitational.io/v2
kind: Cluster
```

Reply:
Fixed in 9bc4135.